src/components/AccountsPage/AccountList/index.js

```diff
@@ -28,16 +28,26 @@ const BodyByMode = {
   list: ListBody,
 }
 
-const matchesSearch = (search: string, account: Account | TokenAccount): boolean =>
-  !search ||
-  (account.type === 'Account'
-    ? `${account.currency.ticker}|${account.currency.name}|${account.currency.ticker}|${
-        account.name
-      }`
-    : `${account.token.ticker}|${account.token.name}`
-  )
-    .toLowerCase()
-    .includes(search.toLowerCase())
+export const matchesSearch = (
+  search?: string,
+  account: Account | TokenAccount,
+  subMatch: boolean = false,
+): boolean => {
```

Comment:
great! can you later add it in live-common?

Reply:
sure :)